logbook-core/src/main/java/org/zalando/logbook/core/DefaultHttpLogFormatter.java

```diff
@@ -53,6 +53,13 @@ public String format(final Precorrelation precorrelation, final HttpRequest requ
         result.append('\n');
 
         writeHeaders(request.getHeaders(), result);
+
+        if (!request.getAttributes().isEmpty()) {
+            result.append("Attributes: ");
+            result.append(request.getAttributes());
```

Comment:
This relies on both the `toString` of the underlying map as well as the `toString` of the values.
Why not e.g. add each attribute on a separate line, maybe prefixed by "Attribute ", so that it reads:

```
Attribute Subject: me
Attribute Source: over there
```

Reply:
(Maybe you come up with another format that is easier on the eyes than `Map.toString`?

Reply:
Fixed in e8c4248.